src/constants/WalletAndCurrencyConstants.js

```diff
@@ -212,26 +212,15 @@ export const getSpecialCurrencyInfo = (currencyCode: string): SpecialCurrencyInf
   }
 }
 
-export const getPrivateKeySweepableCurrencies = (): string[] => {
-  const sweepableCurrencies = []
+export const getCurrenciesSelectedInfo = (info: 'isPrivateKeySweepable' | 'isCustomTokensSupported'): string[] => {
```

Comment:
You don't need this function, just use filter directly instead (replace the 'info' with whatever you need in the scene):
```js
Object.keys(SPECIAL_CURRENCY_INFO).filter(currencyCode => SPECIAL_CURRENCY_INFO[currencyCode]['info'] != null)
```